cmd/restic/integration_test.go

```diff
@@ -1829,3 +1829,53 @@ func TestDiff(t *testing.T) {
 		rtest.Assert(t, r.MatchString(out), "expected pattern %v in output, got\n%v", pattern, out)
 	}
 }
+
+type writeToOnly struct {
+	rd io.Reader
```

Comment:
`io.Reader` would suffice...

Reply:
The `io.Reader` should only be accessed via `WriteTo`. The `Read` method of `rd` must not be accessible.